framework/src/modules/README.md

```diff
@@ -0,0 +1,130 @@
+# Modules
+
+### Table of contents
+
+- [Description](#description)
+- [Custom Modules](#custom-modules)
+- [Custom Assets](#custom-assets)
+
+## Description
+
+Modules are logics to define state changes which will be executed on-chain meaning that it will be a part of the blockchain protocol.
```

Comment:
```suggestion
Modules are logic which define state changes that are executed on-chain, meaning that it will be a part of the blockchain protocol.
```